### ISSUE TITLE
[Feature] Allow more general ratio metrics

### ISSUE THREAD

**Issue by lukesonnet** [growthbook/growthbook#1293]
### Description of Feature

Currently, all ratio metrics are treated as funnel metrics of a sort. We only sum numerator values for users that have a non-zero denominator value.

We should allow users to create general ratio metrics which can count numerator values even if the user has no denominator values.

**Comment by sisungpa** [comment]
Hey @lukesonnet ! Appreciate that this is added to the to-do list 🙌 Just wondering if there is any movement on this issue? This seems to be relevant for anyone interested in a calculating a "per order" metric, which I assume applies to all ecommerce players:)

**Comment by lukesonnet** [comment]
@sisungpa . While it is on the to-do list and it is soon to come up for implementation, I'm not sure it blocks "per order" metrics in general, so long as the numerator is something that is only calculated for those who place orders?

But if you are thinking of more general metrics, then yes this is blocking. I'd be curious to hear your use case just to clarify, though!

**Comment by lukesonnet** [comment]
As of #1573,  All ratio metrics where the denominator is not a binomial now work in this fashion, including all ratio metrics built using Fact Tables.

**Comment by sisungpaV2** [comment]
Thanks @lukesonnet! And sorry for the late reply. I think we discovered that our main challenge around this revolved around having metrics that were not necessarily derived from exactly the same underlying tables and/or timestamps that didn't fully align across numerator and denominator. So we managed to mitigate some of this and now with the more recent update I believe this is working for most of our use cases! 🙌
